### ISSUE TITLE
Devices compatible with the API

### ISSUE THREAD

**Issue by zic10** [huamitech/rest-api#3]
Hi, 
In the documentation is mentioned that is possible access to the user activity data tracked with this wearables: Mi Band series, Amazfit ARC / PACE / EQUATOR / MOONBEAM.

My question is... Is possible access to the data tracked with the Amazfit Bip using the Huami Web API? (activity, heart rate... and GPS?...)

Thanks!
Best Regards

**Comment by 18813131031** [comment]
yes all device is avalible,but each partner Id only can request one main type of device (band ,watch)

**Comment by fpoveda** [comment]
It's possible to access the data tracked with the Amazfit Stratos+ using the Huami Web API?

Thanks!

**Comment by mxr2-a11y** [comment]
> 
> 
> It's possible to access the data tracked with the Amazfit Stratos+ using the Huami Web API?
> 
> Thanks!

+1 for Stratos 3
